src/poetry/console/commands/source/show.py

```diff
@@ -18,42 +19,87 @@ class SourceShowCommand(Command):
         ),
     ]
 
+    def render_source(self, source: Source) -> None:
+        bool_string = {
+            True: "yes",
+            False: "no",
+        }
+
+        table = self.table(style="compact")
+        rows = [
+            ["<info>name</>", f" : <c1>{source.name}</>"],
+            ["<info>url</>", f" : {source.url}"],
+            [
+                "<info>default</>",
+                f" : {bool_string.get(source.default, False)}",
```

Comment:
Shouldn't this `False` be `"no"`? And the same in line 38?

Reply:
`False` is the default value if `source.default` is `None`.

Reply:
ok, but if I understand this right, we want to display `default: yes` or `default: no`. With `False`, it will be `default: False`.

Reply:
Ah, fair point... `bool_string.get(bool(source.default))` might lead to the desired behavior.